aspnetcore/signalr/background-services.md

```diff
@@ -51,7 +53,7 @@ As the `ExecuteAsync` method is called iteratively in the background service, th
 
 ## React to SignalR events with background services
 
-Using the <xref:signalr/dotnet-client>, a `BackgroundService` or `IHostedService` implementor can also be used to connect to SignalR Hubs and respond to events, much like HTML or .NET client apps do using the JavaScript and .NET clients for SignalR.
+Like an HTML using the JavaScript client for SignalR or a .NET desktop app can do using the using the <xref:signalr/dotnet-client>, a `BackgroundService` or `IHostedService` implementation can also be used to connect to SignalR Hubs and respond to events.
```

Comment:
I suggest changing this first sentence to begin with "Like a Single Page App using..." instead of "Like an HTML using...".